statemachine/src/main/java/com/amazonaws/stepfunctions/cloudformation/statemachine/DefinitionProcessor.java

```diff
@@ -79,20 +79,26 @@ private static int getNumDefinitionsInModel(final ResourceModel model) {
      * @param metricsRecorder The MetricsRecorder object used for collecting anonymous property usage metrics
      */
     public static void processDefinition(final AmazonWebServicesClientProxy proxy, final ResourceModel model, final MetricsRecorder metricsRecorder) {
-        if (model.getDefinitionS3Location() != null) {
-            model.setDefinitionString(fetchS3Definition(model.getDefinitionS3Location(), proxy, metricsRecorder));
+        String definition = "";
```

Comment:
Nitpick: you don't need to initialize `definition` to be an empty string, you can just declare it and leave it uninitialized.